From 97827295202d8da3dece5ab1fe5e44c2d0c366b8 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 1 Feb 2023 14:55:30 +0100
Subject: [PATCH] Fix build with gcc-13

* fixes:
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.cc:32:5: error: class 'mediascanner::MediaFile' does not have any field named '_size'
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.cc:33:5: error: class 'mediascanner::MediaFile' does not have any field named '_createdTime'
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.cc:34:5: error: class 'mediascanner::MediaFile' does not have any field named '_modifiedTime'
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:119:5: error: 'uint64_t' does not name a type
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:121:5: error: 'uint64_t' does not name a type
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:122:5: error: 'uint64_t' does not name a type
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:39:5: error: 'uint64_t' does not name a type
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:42:5: error: 'uint64_t' does not name a type
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:43:5: error: 'uint64_t' does not name a type
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:77:18: error: 'uint64_t' has not been declared
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:77:36: error: '_size' was not declared in this scope; did you mean 'ssize_t'?
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:80:25: error: 'uint64_t' has not been declared
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:80:43: error: '_createdTime' was not declared in this scope; did you mean 'setCreatedTime'?
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:81:26: error: 'uint64_t' has not been declared
  mediaindexer/0.1.0-14+gitAUTOINC+a3dc4ce321-r0/git/src/MediaFile.hh:81:44: error: '_modifiedTime' was not declared in this scope; did you mean 'setModifiedTime'?

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending
---
 src/MediaFile.hh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/MediaFile.hh b/src/MediaFile.hh
index f226c79..2ef7733 100644
--- a/src/MediaFile.hh
+++ b/src/MediaFile.hh
@@ -20,6 +20,7 @@
 #ifndef MEDIAFILE_HH
 #define MEDIAFILE_HH
 
+#include <cstdint>
 #include <string>
 #include <list>
 
